psl/psl-core/src/configuration/datasource.rs

```diff
@@ -26,7 +26,7 @@ pub struct Datasource {
     /// In which layer referential actions are handled.
     pub relation_mode: Option<RelationMode>,
     /// _Sorted_ vec of schemas defined in the schemas property.
-    pub schemas: Vec<(String, Span)>,
+    pub namespaces: Vec<(String, Span)>,
```

Comment:
For this field, I agree the name is confusing. On the other hand, it was meant to match the `schemas` property in the datasource, so I feel like we may want them to match. Maybe something to raise to product.

```prisma
datasource abc {
   schemas = ...
   // ^^^^^^^^
}
```